src/server/svr_func.c

```diff
@@ -1155,34 +1136,16 @@ unset_license_location(void)
 		if (pbs_licensing_license_location[0] != '\0') {
 
 			close_licensing();
-			sockets_reset();
 			unlicense_socket_licensed_nodes();
-
-		} else { /* from no license server */
+			clear_license_info();
```

Comment:
I think set_license_location() and unset_license_location() should be moved to license_client.c
I'll leave it up to you to handle it now or later.

Reply:
We can handle that later